tecs/tecs.cpp

```diff
@@ -323,7 +323,7 @@ void TECS::_update_throttle_setpoint(const float throttle_cruise, const matrix::
 		// Assume induced drag scales linearly with normal load factor.
```

Comment:
This comment needs to be amended, eg:

		// Assume induced drag scales with lift coefficient/normal load factor squared.